### DISCUSSION TITLE
Can I run CosmosOS on real hardware?

### DISCUSSION THREAD

**Discussion by fenixthexd** [CosmosOS/Cosmos#1941] [Q&A]
I was wondering that can I run CosmosOS on a real hardware.

**Comment by ilobilo** [comment]
Yes but if you use filesystem there's small chance you could corrupt whole drive so just to be sure turn it off

**Reply by fenixthexd** [reply]
My os doesn't interact with hard drives

**Reply by atlassoftware-cpu** [reply]
how to turn of the filesystem

**Comment by partisani** [comment]
I've Made This

But Is Important To Use Usb Keyboard

![](https://cdn.discordapp.com/attachments/857741677779353632/900920587919183883/image1.jpg)
![](https://cdn.discordapp.com/attachments/857741677779353632/900920588263124992/image0.jpg)

**Reply by coolcoder613eb** [reply]
how?
did you make a bootable usb?

**Reply by Arnotronix75** [reply]
balenaEtcher i think

**Reply by coolcoder613eb** [reply]
i had to use rufus 2.18

**Reply by AsertCreator** [reply]
rufus doesnt work. it errors at me with words "this file is not bootable!!1!1!"

**Reply by AsertCreator** [reply]
p.s. usb keyboards dont work on cosmos, cus the entire usb things isnt supported. however, ps2 works fine

**Reply by coolcoder613eb** [reply]
I don't think Rufus 3 supports bios boot mode ISOs
[Download](https://github.com/pbatard/rufus/releases/download/v2.18/rufus-2.18p.exe) Rufus 2.18

**Comment by Arnotronix75** [comment]
thanks bro

Am So., 27. Nov. 2022 um 04:44 Uhr schrieb Elozor Bruce <
***@***.***>:

**Comment by coolcoder613eb** [comment]
i had to use a old laptop,
if you want to use a newer one
you will have to mess around in the bios
and enable bios boot mode

**Comment by coolcoder613eb** [comment]
[My OS](https://github.com/coolcoder613eb/ebOS)
![IMG_0944](https://user-images.githubusercontent.com/96163908/205465606-1e013912-e05b-49dc-995c-8e24ff5f335d.jpg)

**Comment by Arnotronix75** [comment]
thanks!
